src/pages/components/toggle/usage.mdx

```diff
@@ -69,64 +80,122 @@ user “flips the switch”. They are commonly used for “on/off” switches.
 
 <Caption>Default and small toggle variants</Caption>
 
+## Formatting
+
+### Anatomy
+
+Default and small toggles are both comprised of a component label, a toggle button
+and action text. Default toggles are required to display a component label and
+action text unlike the small toggle where both these elements are optional. The
+small toggle displays a checkmark tick in the on state to ensure the toggle is
+still accessible when label and action text are unavailable.
+
+<Row>
+<Column colLg={8}>
+
+![Anatomy of default and small toggle variants](images/toggle-usage-anatomy.png)
```

Comment:
**Anatomy image**
- The C annotations still look like they are 23px in width instead of 24px. They just look a bit skewed.
- I think you could increase the square around the checkmark tick indicator for the small toggle to be 16px x 16px so its a little larger and more noticeable (example below). That is the size of the current square spec in the production guidelines.

<img width="608" alt="✅ toggle-usage-anatomy" src="https://github.com/user-attachments/assets/31089ef8-1d1a-4754-a485-050fcf262fa1">